projects/stream/index.js

```diff
@@ -26,15 +29,39 @@ async function tvl(_, _1, _2, { api }) {
     target: TBILL_VAULT_CLUB_BOND_CONTRACT,
   });
 
+  const usdc_totalBalance = await api.call({
+    abi: abi.totalBalance,
+    target: LEVUSDC_VAULT_CONTRACT,
+  });
+
+  const wbtc_totalBalance = await api.call({
+    abi: abi.totalBalance,
+    target: HODLWBTC_VAULT_CONTRACT,
+  });
+
+  const weth_totalBalance = await api.call({
+    abi: abi.totalBalance,
+    target: HODLWETH_VAULT_CONTRACT,
+  });
+
+  const usdw4_total = (usdw4_totalSupply - usdw4_vaultBalance) / (vaultToken / baseToken) / 10 ** usdw4_decimals;
```

Comment:
why is `(vaultToken / baseToken)` not applied to the other vaults?

Reply:
usdw4 and all other vaults are different contracts. The other 3 track underlying tokens in contract state directly so simply call a getter that returns amountLocked + current contract balance (funds are held off chain)